packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -156,19 +192,22 @@ class AmountSelectors extends React.Component {
     step: PropTypes.number,
     min: PropTypes.number,
     max: PropTypes.number,
+    maxAmountMessage: PropTypes.string,
```

Comment:
Shouldn't we have a message for the minimum amount as well?

Reply:
I don't think we have a use case today where a min amount message would be useful, so I decided to implement the max amount only

Reply:
I don't get it. The use case is the same for the maximum: The user types a value that is out of the acceptable range.
On the webapp, a pack size with whole increments would have a minimum of 1. If the user types `0,5`, it would not show any message but it will change back to 1 when the user changes focus. They may not notice this change and send the order with an unexpected amount.

Reply:
Yep, you're right.. I missed this point, thanks! I'll submit some changes to handle this

Reply:
done